proposals/0475-tuple-syntax.rst

```diff
@@ -463,6 +469,11 @@ Effect and Interactions
    means either ``Tuple2# Int Bool`` (with ``-XNoListTuplePuns``) or ``Solo# (Tuple2 Int Bool)`` (with
    ``-XListTuplePuns``). Furthermore, ``Tuple# (Int#, Double#)`` would not kind-check with ``-XListTuplePuns``. So we avoid these problems by simply erroring in the 1-element case.
 
+#. The name of the data constructor for unboxed 1-tuples is ``MkSolo#`` rather
+   than ``(# #)`` to distinguish it from the data constructor for 0-tuples.
+   The ambiguity only arises when the name is used unapplied.
+   When applied to an argument, ``MkSolo# a`` can be written and pretty-printed as ``(# a #)``.
```

Comment:
"can be" or "is" (at least for printing)? That is, I would want the prefix `MkSolo#` to be a rare sight for programmers.

Reply:
Updated the text to reflect this preference